pkg/apis/core/types.go

```diff
@@ -475,6 +475,7 @@ type PersistentVolumeReclaimPolicy string
 const (
 	// PersistentVolumeReclaimRecycle means the volume will be recycled back into the pool of unbound persistent volumes on release from its claim.
 	// The volume plugin must support Recycling.
+	// This is deprecated.
```

Comment:
The description is too simple, my suggestion:
'DEPRECATED: The PersistentVolumeReclaimRecycle called Recycle is being deprecated. See announcement here: https://groups.google.com/forum/#!topic/kubernetes-dev/uexugCza84I`

Reply:
Thanks! I pushed a commit with the changes you mentioned.